### PR TITLE
AROS port

### PR DESCRIPTION
**Pull request by BSzili**
Added some fixes to built it for AROS. This time used autotools, not the custom Makefile.

### CODE REVIEW

File: src/picojson.h [hunk 1]
```diff
@@ -210,7 +210,7 @@ namespace picojson {
     if (
 #ifdef _MSC_VER
         ! _finite(n)
-#elif __cplusplus>=201103L || !(defined(isnan) && defined(isinf))
+#elif (__cplusplus>=201103L && !defined(__AROS__)) || !(defined(isnan) && defined(isinf))
 		std::isnan(n) || std::isinf(n)
```
Comment (Ghabry) [inline src/picojson.h]: this one makes we wonder why non-emscripten builds even have picojson included

Comment (carstene1ns) [reply src/picojson.h]: ...because `CreateRequestMapping` is not guarded and generally the async stuff is so ugly nobody wants to touch it ;)

Comment (Ghabry) [reply src/picojson.h]: okay, I will touch it later and then undo that one-line change ;)

**Comment by carstene1ns** [comment]
jenkins: ok to test.

**Comment by carstene1ns** [comment]
jenkins: retest this please.

**Pull request merged**